model/src/main/kotlin/config/PackageConfiguration.kt

```diff
@@ -77,6 +77,7 @@ data class PackageConfiguration(
             is UnknownProvenance -> false
             is ArtifactProvenance -> sourceArtifactUrl != null && sourceArtifactUrl == provenance.sourceArtifact.url
             is RepositoryProvenance -> vcs != null && vcs.matches(provenance)
+            else -> false
```

Comment:
Commit message:
- No simple past like in "Appropriate else cases were added to conditional`when`s." but imperative mood like in "Add `else` cases...".